mos-platform/atari8-stdcart/stdcart.ld

```diff
@@ -0,0 +1,42 @@
+/* Provide imaginary (zero page) registers. */
+__rc0 = 0x80;
+INCLUDE imag-regs.ld
+ASSERT(__rc31 == 0x9f, "Inconsistent zero page map.")
+
+ASSERT(__cart_rom_size == 1 << LOG2CEIL(__cart_rom_size),
+       "CART-ROM size must be a power of 2.")
+
+MEMORY {
+  zp : ORIGIN = __rc31 + 1, LENGTH = 0x100 - (__rc31 + 1)
```

Comment:
Do cartridges really not depend on any of the OS zero page regions? Does any action need to be taken on initialization to disable OS interrupt handlers that might affect this region?